### ISSUE TITLE
Exception when saving page with empty body

### ISSUE THREAD

**Issue by vitorbaptista** [ckan/ckanext-pages#9]
Found in branch ckeditor-blog, commit 0498614dd9

If you save a page with an empty body, we get the stack trace:
```
File '/usr/lib/ckan/default/src/ckan/ckanext/ckanext-pages/ckanext/pages/controller.py', line 266 in pages_show
  self._inject_views_into_page(_page)
File '/usr/lib/ckan/default/src/ckan/ckanext/ckanext-pages/ckanext/pages/controller.py', line 208 in _inject_views_into_page
  root = html.fromstring(_page['content'])
File '/usr/local/lib/python2.7/dist-packages/lxml-3.3.0beta1-py2.7-linux-x86_64.egg/lxml/html/__init__.py', line 704 in fromstring
  doc = document_fromstring(html, parser=parser, base_url=base_url, **kw)
File '/usr/local/lib/python2.7/dist-packages/lxml-3.3.0beta1-py2.7-linux-x86_64.egg/lxml/html/__init__.py', line 600 in document_fromstring
  value = etree.fromstring(html, parser, **kw)
File 'lxml.etree.pyx', line 3010 in lxml.etree.fromstring (src/lxml/lxml.etree.c:64046)
File 'parser.pxi', line 1700 in lxml.etree._parseMemoryDocument (src/lxml/lxml.etree.c:95363)
File 'parser.pxi', line 1571 in lxml.etree._parseDoc (src/lxml/lxml.etree.c:94102)
File 'parser.pxi', line 992 in lxml.etree._BaseParser._parseUnicodeDoc (src/lxml/lxml.etree.c:89871)
File 'parser.pxi', line 579 in lxml.etree._ParserContext._handleParseResultDoc (src/lxml/lxml.etree.c:85606)
File 'parser.pxi', line 680 in lxml.etree._handleParseResult (src/lxml/lxml.etree.c:86732)
File 'parser.pxi', line 630 in lxml.etree._raiseParseError (src/lxml/lxml.etree.c:86199)
XMLSyntaxError: None
```

The problem is at https://github.com/ckan/ckanext-pages/blob/ckeditor-blog/ckanext/pages/controller.py#L205
```python
root = html.fromstring(_page['content'])
```

We can't pass an empty string to ```html.fromstring```.

**Comment by metaodi** [comment]
I think you fixed this bug with 1fcc15dcf7e5f4976d2e5ca73d727a78107b47c5.

**Comment by vitorbaptista** [comment]
@metaodi sorry for not closing it years ago :sweat_smile:
